cmd/nomad-toast/allocations/config.go

```diff
@@ -2,14 +2,23 @@ package allocations
 
 import (
 	"github.com/jrasell/nomad-toast/pkg/config"
+	"github.com/spf13/viper"
 )
 
 func getConfig() (*config.ToastConfig, error) {
 	nomadConfig := config.GetNomadConfig()
+	nomadConfig.AllocCfg = getAllocConfig()
 	slackConfig := config.GetSlackConfig()
 
 	return &config.ToastConfig{
 		Nomad: &nomadConfig,
 		Slack: &slackConfig,
 	}, nil
 }
+
+func getAllocConfig() *config.AllocConfig {
```

Comment:
ideally we could move this to `/pkg/config` and have a new file in there which is called filtering or such, this can then be used to setup the flags and allow the commands to pull the config from viper.

Reply:
I'm not 100% clear on what you mean, are you suggesting `getAllocCfg` should move to `/pkg/config/filtering.go`? But then where does `filtering` get the `cfgKey...` consts from?

Perhaps if you could elaborate ...